test/image/compare_pixels_test.js

```diff
@@ -51,15 +51,31 @@ var QUEUE_WAIT = 10;
 var pattern = process.argv[2];
 var mockList = getMockList(pattern);
 var isInQueue = (process.argv[3] === '--queue');
+var isCI = process.env.CIRCLECI;
 
 if(mockList.length === 0) {
     throw new Error('No mocks found with pattern ' + pattern);
 }
 
 // filter out untestable mocks if no pattern is specified
 if(!pattern) {
-    console.log('Filtering out untestable mocks\n');
+    console.log('Filtering out untestable mocks:');
     mockList = mockList.filter(untestableFilter);
+    console.log('\n');
+}
+
+// gl2d have limited image-test support
+if(pattern === 'gl2d_*') {
+
+    if(!isInQueue) {
+        console.log('WARN: Running gl2d image tests in batch may lead to unwanted results\n');
+    }
+
+    if(isCI) {
+        console.log('Filtering out multiple-subplot gl2d mocks:');
```

Comment:
Maybe this could be a slightly more general log entry b/c the `pointcloud` test is in this set too -- or we might want to alter the `pointcloud` test such that it doesn't test the blending effect (I can do it in a separate small PR) but even then, at least *most* of the `pointcloud` stuff would be tested, vs. no `pointcloud` test on CI right now.